SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2373,3 +2374,14 @@ void USpatialNetDriver::SetSpatialDebugger(ASpatialDebugger* InSpatialDebugger)
 
 	SpatialDebugger = InSpatialDebugger;
 }
+
+FUnrealObjectRef USpatialNetDriver::GetCurrentPlayerControllerRef()
+{
+	USpatialNetConnection* NetConnection = GetSpatialOSNetConnection();
```

Comment:
nit - we'd generally put each of these * definitions in an if statement

Reply:
So nobody here has an aversion to deep-nested if statements ?
`if(USpatialNetConnection* NetConnection = GetSpatialOSNetConnection())
{
  if(APlayerController* PlayerController = Cast<APlayerController>(NetConnection->OwningActor))
  {
    if (PackageMap)
    {
      return PackageMap->GetUnrealObjectRefFromObject(PlayerController);
    }
  }
}`